### PR TITLE
[14.0][MIG] connector_voicent

### PR DESCRIPTION
**Pull request by Vimal-OSI**

### CODE REVIEW

File: connector_voicent/__manifest__.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright (C) 2019 Open Source Integrators
+# Copyright (C) 2021 Open Source Integrators
```
Comment (dreispt) [inline connector_voicent/__manifest__.py]: Don't change the work creation date please.
At best edit to `2019-2021`

**Review by dreispt — CHANGES_REQUESTED** [review]

File: connector_voicent/__manifest__.py (hunk 1 above)
Comment (max3903) [inline connector_voicent/__manifest__.py]:
```suggestion
# Copyright (C) 2019 Open Source Integrators
```

**Comment by max3903** [comment]
@Urvisha-OSI Can you fix the errors please?

**Review by max3903 — APPROVED** [review]

**Comment by Freni-OSI** [comment]
@max3903 It's ready for review. :green_heart: 
cc: @dreispt

**Pull request merged**